pyth-sdk-solana/src/state.rs

```diff
@@ -456,8 +552,8 @@ pub fn load_product_account(data: &[u8]) -> Result<&ProductAccount, PythError> {
 }
 
 /// Get a `Price` account from the raw byte value of a Solana account.
-pub fn load_price_account(data: &[u8]) -> Result<&PriceAccount, PythError> {
```

Comment:
Is it ok that this functions loads the account is PriceAccountSolana ? I think it's used in pyth-agent